src/public/GnosisUniswapTokensList.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "Uniswap Labs Default",
-  "timestamp": "2024-02-26T23:41:27.220Z",
+  "timestamp": "2024-04-08T14:06:46.865Z",
   "version": {
     "major": 11,
-    "minor": 15,
+    "minor": 17,
```

Comment:
Why the version bump without changes?

Reply:
It come from https://gateway.ipfs.io/ipns/tokens.uniswap.org.
We don't get all tokens from the list because not all of then are known in Omnibridge